src/dataflow/src/render/reduce.rs

```diff
@@ -31,8 +31,110 @@ use super::context::Context;
 use crate::render::context::Arrangement;
 use crate::render::datum_vec::DatumVec;
 
+// Top level object for the Reduce lower-level IR
+#[derive(Clone, Debug)]
+enum ReduceExpr {
+    // No aggregations, just distinct
+    Distinct,
+    // Aggregations that can be computed in place
+    Accumulable(AccumulableExpr),
```

Comment:
For me, this is missing some explanation. It is the top-level item in the file, and I can't tell what it is or how it should be used.